.github/workflows/docker.yml

```
@@ -0,0 +1,79 @@
+name: Docker
+
+# This will run when:
+# - when new code is pushed to main/develop to push the tags
+#   latest and develop
+# - when a pull request is created and updated  to make sure the
+#   Dockerfile is still valid.
+# To be able to push to dockerhub, this execpts the following
+# secrets to be set in the project:
+# - DOCKERHUB_USERNAME : username that can push to the org
+# - DOCKERHUB_PASSWORD : password associated with the username
+on:
+  push:
+    branches:
+      - main
+      # TODO: Remove these once it has been merged back to main
+      - ReactionMiner_v2
+      - add-docker-workflow
+
+  pull_request:
+  
+  # Trigger the workflow on release activity
+  release:
+    # Only use the types keyword to narrow down the activity types that will trigger your workflow.
+    types:
+      - published
+      - edited
+      - created
+
+# Certain actions will only run when this is the main repo.
+env:
+  MAIN_REPO: moleculemaker/ReactionMiner
```

Change main repo (I tested with our fork, but we want to watch your repo instead)
```suggestion
  MAIN_REPO: maszhongming/ReactionMiner
```